scenarios/common/registry/registry_nginx.conf

```diff
@@ -0,0 +1,54 @@
+# See https://docs.gunicorn.org/en/latest/deploy.html
```

Comment:
For a large part these `nginx.conf` files are the same, maybe you can create a base .conf file and use [include](http://nginx.org/en/docs/ngx_core_module.html#include) to include it in specific .conf files?

Reply:
You're right. In fact, I think the registry proxy would run fine with the external site configuration :-).

There may be a problem though with the differences being on the inside of the nested structure. I think to really deduplicate it, I would have to have multiple base files for different parts, i.e. one for the top part, one for the top part within the `http` block, and one for the common bits of the second `server` block. I think that would end up being a bit annoying to understand though. Maybe a bit of duplication is not so bad?

Reply:
Check, that sounds more complicated indeed

Reply:
Actually, there's a better solution. I'm already fixing up files here and there with Ansible, so I've done that here as well now. There's a single nginx.conf.tmpl, and Ansible copies that three times and then patches it to create the correct file. That has the disadvantage that the whole Ansible thing is not super-transparent, but it is a bit more compact. I'm going to merge it like this, we can always change it again later if it turns out not to work.